lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1279,26 +1279,29 @@ computeUtxoStatistics btype utxos =
     (^!) = (^)
 
 {-------------------------------------------------------------------------------
-                             Blockchain Parameters
+                              Network Parameters
 -------------------------------------------------------------------------------}
 
--- | Initial blockchain parameters loaded from the application configuration and
--- genesis block.
+-- | Records the complete set of parameters currently in use by the network.
```

Comment:
Actually this isn't the complete set of parameters. Just those which are relevant to the wallet.

Reply:
Fixed in e26d3ae8bce792b5530bd8f82324accd95a0c668.